sdk/python/feast/embedded_go/online_features_service.py

```diff
@@ -132,6 +132,16 @@ def get_online_features(
         resp = record_batch_to_online_response(record_batch)
         return OnlineResponse(resp)
 
+    def start_server(self, host: str, port: int, server_type: str):
```

Comment:
nitpick: maybe just let's have two method start_grpc_server and start_http_server?